File: app/src/main/java/oop1/ex01.java
package oop1;

public class ex01 {
    static class Student {
        String name;
        int age;

        Student(String name, int age) {
            this.name = name;
            this.age = age;
        }

        void printInfo() {
            System.out.println("name=" + name + ", age=" + age);
        }
    }

    public static void main(String[] args) {
        Student student = new Student("Alice", 20);
        student.printInfo();
    }
}

File: app-kotlin/src/main/kotlin/oop1/ex01.kt
package oop1

class ex01 {
    data class Student(val name: String, val age: Int) {
        fun printInfo() {
            println("name=$name, age=$age")
        }
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            val student = Student("Alice", 20)
            student.printInfo()
        }
    }
}
